torch/_dynamo/variables/tensor.py

```diff
@@ -181,8 +181,8 @@ def var_getattr(self, tx, name):
 
         # It's hard to get resize_() on graph input work properly across
         # dynamo/aot/inductor, just fall back.
-        if name in ("resize_", "unsqueeze_") and self.source is not None:
-            # Delay the graph break to the actual call of unsqueeze_/resize_
+        if name in ("resize_", "unsqueeze_", "resize_as_") and self.source is not None:
+            # Delay the graph break to the actual call of unsqueeze_/resize_/resize_as_
```

Comment:
It fixes the common place. So it is also an issue with CUDA, not just CPU?

Reply:
It is also an issue with CUDA that gets wrong results.

Reply:
btw, I saw @eellison suggested to use inplace_view tag check from https://github.com/pytorch/pytorch/pull/99986/files#r1181696824. Seems a more general approach rather than appending more functions here?

Reply:
It seems that resize_as_ doesn't have an inplace_view tag now. I will add support and refactor code in another pr.
https://github.com/pytorch/pytorch/blob/c91a41fd6827f076224477913b1c09d20a887935/aten/src/ATen/native/native_functions.yaml#L6362-L6367